CHANGES

```diff
@@ -1,22 +1,10 @@
 Changes from V8.5 to V8.6beta1
 ==============================
 
-Bugfixes
+Kernel
 
-- #4527: when typechecking the statement of a lemma using universe polymorphic
-  definitions with explicit universe binders, check that the type can indeed be
-  typechecked using only those universes (after minimization of the other,
-  flexible universes), or raise an error (fixed scripts can be made forward
-  compatible).
-- #4726: treat user-provided sorts of universe polymorphic records as rigid
-  (i.e. non-minimizable).
-- #4592, #4932: notations sharing recursive patterns or sharing
-  binders made more robust.
-- #4780: Induction with universe polymorphism on was creating ill-typed terms.
-- #3070: fixing "subst" in the presence of a chain of dependencies.
-- When used as an argument of an ltac function, "auto" without "with"
-  nor "using" clause now correctly uses only the core hint database by
-  default.
+- A new, state-of-the-art universe constraint checker that
+  can outperform the previous version by an order of magnitude.
```

Comment:
Why "can"?

Being more explicit that "outperform" is about the efficiency/complexity of the algorithm?